.github/workflows/publish.yml

```diff
@@ -27,3 +27,10 @@ jobs:
         uses: DEFRA/cdp-build-action/build@main
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
You need to prefix it with this line
`- run: npm install && npm test`
so there is coverage data to scan. 
See https://github.com/DEFRA/cdp-example-node-backend/blob/main/.github/workflows/publish.yml#L30